vnext/Microsoft.ReactNative/Views/Image/ReactImage.cpp

```diff
@@ -219,9 +222,15 @@ winrt::fire_and_forget ReactImage::SetBackground(bool fireLoadEndEvent) {
     if (strong_this && fireLoadEndEvent) {
       strong_this->m_onLoadEndEvent(*strong_this, false);
     }
+    co_return;
```

Comment:
Separate issue, and didn't repro, but it seems like we want probably want to bail out if there is some exception here.